src/pages/posts/[slug]/index.astro

```diff
@@ -39,3 +40,4 @@ const pagination = getPagination({
 ---
 
 {post ? <PostDetails post={post} /> : <Posts {...pagination} />}
+@utils/postFilters
```

Comment:
What are these?
When I checkout this PR and run the project without these, it still working fine.

Reply:
In a lot of places around the code you have a filter for the array that checks if the draft flag is false. I moved that logic to one place so it was easy to change, keeping the code DRY

Reply:
Yes, I get that.
But what does line number 43 do?
In that file, `draftFilter` is already used in line number 15.
So, I don't think `@utils/postFilters` at line 43 is necessary.

Reply:
Ahh sorry, I replied to the comment without seeing it in context of the file. Not sure. That might be VSCode trying to be helpful

Reply:
Yes. 
When I rename some imports, VSCode tried to add those things.
I removed those in my new commit.